src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -141,7 +141,6 @@ static DataTestUtility()
                 if (!string.IsNullOrEmpty(TCPConnectionStringNoneVBS))
                 {
                     AEConnStrings.Add(TCPConnectionStringNoneVBS);
```

Comment:
Accidentally is removed!
```suggestion
                    AEConnStrings.Add(TCPConnectionStringNoneVBS);
                    AEConnStringsSetup.Add(TCPConnectionStringNoneVBS);
```

Reply:
That is the purpose of the PR.